server/service/src/sync/api/common_records.rs

```diff
@@ -61,7 +61,7 @@ impl SyncActionV5 {
 }
 
 #[derive(Error, Debug)]
-#[error("Failed to parse V5 remote record into sync buffer row: {source:?} {record:?}")]
+#[error("Failed to parse V5 remote record into sync buffer row, record: '{record:?}'")]
```

Comment:
This is an example where source doesn't need to be specified, it will still be printed by format_error, if error enum variant or error struct has 'source' field or annotation of`#[source]` or `#[from]`